morphia/src/test/java/org/mongodb/morphia/converters/ConverterTest.java

```diff
@@ -17,10 +17,11 @@
 package org.mongodb.morphia.converters;
 
 import org.junit.Assert;
+import org.mongodb.morphia.TestBase;
 
 import static java.lang.String.format;
 
-class ConverterTest<F, T> {
+class ConverterTest<F, T> extends TestBase {
```

Comment:
You probably should also add round trip test cases for your other ConverterTests to prevent future regressions.